packages/core/base/src/models/events.ts

```diff
@@ -0,0 +1,11 @@
+export enum CheckoutEvents {
```

Comment:
This type can be imported in crossbit-main and used so there is a single source of truth for both customers and us